recipes/imgui/all/test_package/my_imgui_config.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
```

Comment:
I don't get why it should be so complicated. why can't I just include that header file? is it really needed to inject things inside the library?

Reply:
It is how `imgui` works 🤷  Nothing to say here